desktop/apps/editorial_features/routes.coffee

```diff
@@ -55,3 +55,9 @@ Articles = require '../../collections/articles.coffee'
   target = 'https://' + VANITY_BUCKET + '.s3.amazonaws.com' + '/' + req.params[0]
   proxy.web req, res, target: target, (err) ->
     res.redirect 301, '/articles' if err
+
+setVideoIndex = (curation, slug, res) ->
+  for section, i in curation.get 'sections'
+    if section.slug is slug
+      return i
+  return res.redirect 301, '/venice-biennale'
```

Comment:
This will break - we want to do this after the `success`, not while finding a video index. This wouldn't return a real videoIndex so it would break the render if we place it here.